### ISSUE TITLE
{id} placeholder

### ISSUE THREAD

**Issue by l3nnartt** [tebexio/Tebex-Minecraft#30]
Since I didn't get a reply in the original issue, I'm creating a new one.
The problem also exists with 2.0.2

https://github.com/tebexio/Tebex-Minecraft/issues/22

**Comment by WildBamaBoy** [comment]
Confirmed, yes, this will be resolved in 2.0.3 by 6834c99 and 0551db6

**Comment by l3nnartt** [comment]
```
[21:16:00 INFO] [Tebex]: Dispatching command 'executebuycraftcommand {id} *****' for player 'L3nnart_'.
```
the issue still exists in 2.0.3
